src/components/AudioPlayer/PlayerButtons/Shuffle/ShuffleButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { setRandom } from 'store/actions/playerActions';
+import { ShuffleIcon } from 'components/icon';
+import { PlayerButton } from 'style/button';
+import { gray, orange } from 'style/colors';
+
+const ShuffleButton = () => {
+  const onRandom = useSelector((state) => state.player.get('onRandom'));
+  const dispatch = useDispatch();
+
+  const strokeColor = onRandom ? orange : gray;
+
+  return (
+    <PlayerButton onClick={() => dispatch(setRandom(!onRandom))}>
```

Comment:
You should not use `dispatch` in presentational components. Please move this into a container. All these actions belong to the audio player, not the buttons.